scripts/render_all_coursera.R

```diff
@@ -26,25 +20,32 @@ if (length(old_files) > 0) {
   file.remove(old_files)
 }
 
-# Copy over the assets
-# fs::dir_copy("assets", file.path(output_dir, "assets"))
+# Copy these directories over if they don't exist in the output folder
+needed_directories <- c("assets", "code_output", "resources")
 
-# Set up function which will call the
-render_coursera <- function(rmd_file, verbose = FALSE) {
+lapply(needed_directories, function(needed_dir) {
+  if (!dir.exists(file.path(output_dir, needed_dir))) {
+    fs::dir_copy(needed_dir, file.path(output_dir, needed_dir), overwrite = TRUE)
+  }
+})
 
-  # Build the command
-  r_command <-
-    paste0("Rscript --vanilla ", file.path(root_dir, "scripts", "render_rmd_coursera.R"),
-    " --rmd ", file.path(root_dir, rmd_file),
-    " --css_file ", file.path("assets", "style_ITN_coursera.css"),
-    " --style")
+# Slightly different path for the libs folder
+libs_path <- file.path("docs", "libs")
+if (!dir.exists(file.path(output_dir, "libs"))) {
+  fs::dir_copy(libs_path, file.path(output_dir, "libs"), overwrite = TRUE)
+}
 
-  if (verbose) {message(r_command)}
+# Retrieve list of Rmd files from the _bookdown.yml
+output_yaml <- yaml::yaml.load_file(file.path(root_dir, "_output.yml"))
 
-  # Run it!
-  system(r_command)
+# Change CSS file to coursera special one
+output_yaml$`bookdown::gitbook`$css <- gsub("\\.css", "_coursera.css", output_yaml$`bookdown::gitbook`$css)
```

Comment:
Here's where I'm changing the output file to be the coursera yaml which has the shrink the toc lines. @carriewright11

Reply:
ah yes, I see it now